packages/babel-preset-typescript/src/plugin-rewrite-ts-imports.ts

```diff
@@ -0,0 +1,22 @@
+import { declare } from "@babel/helper-plugin-utils";
+import type { types as t } from "@babel/core";
+import type { NodePath } from "@babel/traverse";
+
+export default declare(function () {
+  return {
```

Comment:
Can you add a `name` property? It is good for serializing the config when `showConfig` is enabled.